taichi/codegen/codegen_llvm.h

```diff
@@ -72,8 +72,7 @@ class CodeGenLLVM : public IRVisitor, public LLVMModuleBuilder {
   std::vector<OffloadedTask> offloaded_tasks;
   BasicBlock *func_body_bb;
 
-  std::unordered_map<OffloadedStmt *, std::vector<llvm::Value *>>
-      offloaded_loop_vars_llvm;
+  std::unordered_map<Stmt *, std::vector<llvm::Value *>> loop_vars_llvm;
```

Comment:
nit: is it possible to use `const Stmt*` here? i don't think the key is modified?